doc/reference/config-options.rst

```diff
@@ -30,6 +30,10 @@ Integer
   first character must not be a ``0``. Only positive values are
   expected.
 
+String
+  A sequence of UTF-8 characters.  String-length limits or specific
```

Comment:
AFAIK we only expect strings in standard ASCII rather than UTF-8.

Reply:
UTF-8 would be better than simply ASCII characters here, just in case we needed to account for other languages in the future and allowing for line-drawing characters and such that aren't available in ASCII-only text.  We can review that after the merge if needed.